config/rbac/role.yaml

```diff
@@ -22,7 +28,6 @@ rules:
   - list
   - update
   - watch
-  - patch
```

Comment:
why did you remove path?

Reply:
> why did you remove patch?

It vanished by accident when executing controller-gen. I'll re-add it in the controller and role.